tsl/src/remote/dist_commands.c

```diff
@@ -427,18 +429,36 @@ ts_dist_cmd_close_prepared_command(PreparedDistCmd *command)
 	list_free_deep(command);
 }
 
+static const char *
+get_procedure_name(FunctionCallInfo fcinfo)
```

Comment:
this is copy of postgres get_func_name, you should use that instead

Reply:
Ah, didn't see it. Will remove.

Reply:
Removed